website_menu_permission/README.rst

```diff
@@ -0,0 +1,83 @@
+=======================
+Website Menu Permission
+=======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fwebsite-lightgray.png?logo=github
+    :target: https://github.com/OCA/website/tree/11.0/website_menu_permission
+    :alt: OCA/website
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/website-11-0/website-11-0-website_menu_permission
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/186/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Show/hide website menu items by user group.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Website ==> Configuration ==> Website menu's.
+#. Edit the menu that you want to limit to certain groups, and add
+   the names of the groups that should have access.
```

Comment:
This explanation is confusing because:

- In Website -> Configuration there are 2 different menu entries for the same model website.menu
- You need developer mode in order to see "Website menu's" entry. The other entry "Menu" you can see without developer mode.

I think is enough one menu entry and not two.

Reply:
I only have one menu entry for website menu's. And I am in developer mode. If you see a second option for website menu's can you check from what module that is installed?

Reply:
that's runbot installing all modules from the repository. I see xmlid `website_multi_theme.action_configure_menu_website`